components/settings/AuthConnectList.vue

```diff
@@ -169,7 +171,7 @@ export default {
 }
 
 .l-button {
-  @each $platform in google facebook twitter {
+  @each $platform in google facebook twitter matters {
     &--#{$platform} {
       background-color: map-get($platform-color-map, $platform);
```

Comment:
does `$platform-color-map have` `matters` in `~/assets/_variables.scss`?

Reply:
no, not sure what should it be

Reply:
use `white` then